dist/weather-card.js

```diff
@@ -150,10 +151,10 @@ class WeatherCard extends LitElement {
     return html`
       <ha-card @click="${this._handleClick}">
         ${this._config.current !== false ? this.renderCurrent(stateObj) : ""}
-        ${this._config.details !== false ? this.renderDetails(stateObj, lang) : ""}
+        ${this._config.details !== false ? this.renderDetails(stateObj, lang, time_format) : ""}
```

Comment:
We should pass `hass.locale` here

Reply:
That can also work, but in HA there is a time format setting that can override system locale . If it's set, i think it should use time_formwt, then fallback to locale, and then to hass.lang.

Reply:
would be nice if this can be merged. I use an english OS, dutch locale. Logbook and history shows 24hr timeformat, unfortunately this card does not and gives me AM/PM times. It gives me headaches 🤣